docs/spec/v1/gitrepositories.md

```diff
@@ -103,8 +103,8 @@ Git repository.
 
 **Note:** Unlike using `git`, the
 [shorter scp-like syntax](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol)
-is not supported for SSH addresses (e.g. `user@example.com:repository.git`).
-Instead, the valid URL format is `ssh://user@example.com:22/repository.git`.
+is not supported for SSH addresses (e.g. gitlab.com will give you this pattern when you clone a project `ssh://git@gitlab.com:root_group/a_group/your_repo.git`).
```

Comment:
You are right, I am rollbacking. It was just for explaining that if you have a second ":" like this `ssh://git@gitlab.com:root_group/group/repo.git` then it will not work, you need to replace the second by "/" like this `ssh://git@gitlab.com/root_group/group/repo.git` (I faced this usecase)